UKHomeOffice/cerberus-service
# COP-11006: Add test for Update cop-targeting-api target information sheet for RoRo freight mode split

## Description
Add test for Update cop-targeting-api target information sheet for RoRo freight mode split
## To Test
npm run cypress:test:local -- --spec cypress/integration/create-tasks.spec.js

Should verify mode and modeCode for RORO-Unaccompanied frieght from target information sheet

Should verify mode and modeCode for RORO-Accompanied frieght from target information sheet
## Developer Checklist

\* Required
